oops/hosts/juno/jiram/img.py

```
@@ -220,27 +216,16 @@ def initialize(time, ck='reconstructed', planets=None, asof=None,
-            gapfill     True to include gapfill CKs. False otherwise.
-            mst_pck     True to include MST PCKs, which update the rotation models
-                        for some of the small moons.
-            irregulars  True to include the irregular satellites;
-                        False otherwise.
+            asof        Only use SPICE kernels that existed before this date;
+                        None to ignore.
+            kwargs:     Arguments for juno.initialize() and Body.define_solar_system()
         """
 
         # Quick exit after first call
         if IMG.initialized: return
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)